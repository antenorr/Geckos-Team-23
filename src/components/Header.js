import React, {Component} from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
    <div id="header" className="header">
        OneTab++
        <ul className="ul-no-bullet header-navigation">
            <li >
                <Link to={`/settings`}>Settings</Link>
            </li>
        </ul>
    </div>
);

export default Header;
