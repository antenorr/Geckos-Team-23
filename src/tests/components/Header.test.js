/**
 * @jest-environment node
 */ 

import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Header from '../../components/Header';

test('should render Header correctly', () => {
  const wrapper = shallow(<Header />);
  expect(toJSON(wrapper)).toMatchSnapshot();
  // FYI: expect(wrapper.find('h1').text()).toBe('OneTab++!');
});
