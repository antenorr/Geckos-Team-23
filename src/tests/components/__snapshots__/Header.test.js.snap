// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="header header-bar"
  id="header"
>
  <div
    className="header-brand"
  >
    OneTab++
  </div>
  <div>
    <ul
      className="ul-no-bullet"
    >
      <li
        className="header-li"
      >
        <Link
          replace={false}
          to="/home"
        >
          Home
        </Link>
      </li>
      <li
        className="header-li"
      >
        <Link
          replace={false}
          to="/settings"
        >
          Settings
        </Link>
      </li>
    </ul>
  </div>
</div>
`;
